nesta/core/routines/arxiv/arxiv_grid_task.py

```diff
@@ -171,8 +172,8 @@ def run(self):
             if not count % 100:
                 logging.info(f"{count} processed articles from {total} : {(count / total) * 100:.1f}%")
 
-            if self.test and count == 50:
-                logging.warning("Exiting after 50 articles in test mode")
+            if self.test and count == 5000:
```

Comment:
Needed a larger chunk of articles to fully validate that all my institutes were being matched